boards/FSM/Src/main.c

```diff
@@ -44,6 +44,7 @@
 #include "App_FsmWorld.h"
 #include "App_InRangeCheck.h"
```

Comment:
```suggestion
```

Reply:
On the note I mentioned here, this should've been removed too

Reply:
fixed